index.html

```diff
@@ -231,45 +231,44 @@
           };
         </pre>
         <p>
-          When the <code><dfn for='ScreenOrientation'>lock</dfn>()</code>
-          method is invoked, the <a>user agent</a> MUST run the <a>apply an
-          orientation lock</a> steps to the <a>responsible document</a> using
+          When the <dfn>lock</dfn>() method is invoked,
+          the <a>user agent</a> MUST run the <a>apply an orientation lock</a>
+          steps to the <a>responsible document</a> using
           <var>orientation</var>.
         </p>
         <p>
-          When the <code><dfn for='ScreenOrientation'>unlock</dfn>()</code>
-          method is invoked, the <a>user agent</a> MUST run the steps to
-          <a>lock the orientation</a> of the <a>responsible document</a> to the
+          When the <dfn>unlock</dfn>() method is
+          invoked, the <a>user agent</a> MUST run the steps to <a>lock the
+          orientation</a> of the <a>responsible document</a> to the
           <a>responsible document</a>'s <a>default orientation</a>.
         </p>
         <p class='note'>
-          <a data-link-for='ScreenOrientation' data-lt='unlock'>unlock()</a> does not
-          return a promise because it is equivalent to locking to the
+          <a data-lt='unlock'>unlock()</a>
+          does not return a promise because it is equivalent to locking to the
           <a>default orientation</a> which might or might not be known by the
           <a>user agent</a>. Hence, the <a>user agent</a> can not predict what
           the new orientation is going to be and even if it is going to change
           at all.
         </p>
         <p>
-          When getting the <code><dfn for='ScreenOrientation'>type</dfn></code>
-          attribute, the <a>user agent</a> MUST return the <a>responsible
-          document</a>'s <a>current orientation type</a>.
+          When getting the <dfn>type</dfn> attribute,
+          the <a>user agent</a> MUST return the <a>responsible document</a>'s
+          <a>current orientation type</a>.
         </p>
         <p>
-          When getting the <code><dfn for=
-          'ScreenOrientation'>angle</dfn></code> attribute, the <a>user
-          agent</a> MUST return the <a>responsible document</a>'s <a>current
-          orientation angle</a>.
+          When getting the <dfn>angle</dfn> attribute,
+          the <a>user agent</a> MUST return the <a>responsible document</a>'s
+          <a>current orientation angle</a>.
         </p>
         <p>
-          The <code><dfn for='ScreenOrientation'>onchange</dfn></code>
-          attribute is an <a>event handler</a> whose corresponding <a>event
-          handler event type</a> is <code>change</code>.
+          The <dfn>onchange</dfn> attribute is an
+          <a>event handler</a> whose corresponding <a>event handler event type</a>
+          is <code>change</code>.
```

Comment:
we should probably write `<code>"change"</code>`... if we defined "change", we should probably link to it... can't remember tho.